src/beman/exemplar/CMakeLists.txt

```diff
@@ -31,31 +34,31 @@ if(BEMAN_EXEMPLAR_INSTALL_CONFIG_FILE_PACKAGE)
     include(CMakePackageConfigHelpers)
 
     configure_package_config_file(
-        "${PROJECT_SOURCE_DIR}/cmake/${PROJECT_NAME}-config.cmake.in"
-        "${PROJECT_BINARY_DIR}/cmake/${PROJECT_NAME}/${PROJECT_NAME}-config.cmake"
-        INSTALL_DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME}"
+        "${CMAKE_CURRENT_SOURCE_DIR}/beman.exemplar-config.cmake.in"
```

Comment:
I'm in favor of use before reuse, so I don't have reason not to do this. 
Is there a technical reason not to use the PROJECT_NAME defined in https://github.com/bemanproject/exemplar/blob/fb698c289ea060c2ef64b98486dd2cb1618596b7/CMakeLists.txt#L6
at the top-level CML.txt? 
It does seem a little fragile, and might not compose well?